docs/README.adoc

```diff
@@ -134,18 +134,6 @@ https://github.com/redhat-service-assurance/telemetry-framework/archive/<release
 the host level of each node in your cluster prior to instantiating an SAF
 instance if ElasticSearch is intended to be deployed.
 
-ifdef::target-upstream[]
```

Comment:
:-1: Please be specific about why this should be removed

Reply:
I should have been more clear. The `quickstart` script does not handle the [creation of manifests](https://github.com/redhat-service-assurance/telemetry-framework/tree/master/docs#generating-the-manifests-for-saf) now required for the events feature Leif just added. In the documentation, the section mentioning the script comes before the section mentioning the manifests, implying the script handles the creation of these. On a first time install, this will result in the Prometheus operator and every Elasticsearch pod to fail to create because the yaml files do not exist. I ran into this issue when launching SAF on Minishift yesterday. 

If we wanted to keep the script in the docs, the section mentioning it should be moved to the [deployment section](https://github.com/redhat-service-assurance/telemetry-framework/tree/master/docs#deploying-saf-to-the-openshift-environment) of the docs (which again, occurs after mention of the script), and the section adjusted to accommodate using it. I suggested removing it because, once a user reaches this section, the script only automates two of the deployment steps and the user having to do three is not that much bigger of a deal.

Reply:
Upon further investigation it looks like it is supposed to generate the manifests but did not in my case. I will explore further why this was the case